### PR TITLE
Updating "Suggest relevant content" section

### PR DESCRIPTION
**Pull request by litkaemily**
Updating the "Firefox by default shares data to ... Suggest relevant content" section, specifically to reflect that Pocket will now be passing the client's language and country location information to Pocket.

cc @peiying2

### CODE REVIEW

**Comment by litkaemily** [comment]
Hi, thanks for looking! I still don't have access - I think actually you or Michael have to grant it to me. But, yes, if you could please merge, that'd be great!

**Comment by peiying2** [comment]
> Hi, thanks for looking! I still don't have access - I think actually you or Michael have to grant it to me. But, yes, if you could please merge, that'd be great!

I am a user, not an admin for this, so I can't change other's access status. I will merge this, not a problem at all. I hope you filed the bug for github to change your current status. 

Several questions I have:
1). With this change, do you need to revise the publishing date of the PN? 
2). How soon do you need to get this pushed to production? We can push it out today if you want after verifying it on staging
3). Do you want to have the localized version updated? If so, when do you need this by?

**Comment by litkaemily** [comment]
Thanks! I filed the bug and they told me the following "Emily - access to repositories is handled by the repository admins, not by the "owners". In this case either Michael or Peiying need to add you."

Re your questions: 
1. Yes! Thank you, I'll do this now. 
2. Today would be great, but this week is fine. 
3. Yes, we'll need to localize this. What is the typical time-frame for this?

**Comment by litkaemily** [comment]
OK, i believe i made the update in this PR. Please let me know if I didn't do it correctly!

File: firefox_privacy_notice/en-US.md [hunk 1]
```diff
@@ -35,7 +35,9 @@ Firefox displays content, such as “Snippets” (messages from Mozilla), Add-on
...
+* __Webpage, Language, and Location data for Pocket Recommendations__: We recommend content to you based on your browsing history, language, and country location. The process of deciding which stories you should see based on your browsing history happens locally in your copy of Firefox, and neither Mozilla nor Pocket receives a copy of your browsing history. To help you see relevant Pocket Recommendations based on your location, Firefox shares your language and country location with Pocket.
+
+    Mozilla and Pocket receive aggregated data about the recommendations you see and click. We also share aggregated data about the sponsored content you see and    click with our third-party ad platform Adzerk so advertisers can see how many people click on their articles. This aggregated data does not identify you personally.
```
Comment (peiying2) [inline firefox_privacy_notice/en-US.md:40]: Remove extra white space between `and` and `click with our third-party...`

Comment (litkaemily) [reply firefox_privacy_notice/en-US.md]: Good catch! It should be updated now

**Comment by peiying2** [comment]
@litkaemily please review the updates from both docs on staging. Once you are ready (everything looks good), please create the PR for production push for review.

**Pull request merged**